frameworks/helloworld/tests/test_mount_volumes.py

```diff
@@ -54,7 +102,7 @@ def test_kill_node():
 
 
 @pytest.mark.sanity
-def test_kill_agent():
+def test_kill_agent(mount_volumes):
```

Comment:
@mellenburg: Is adding this fixture to both tests going to add mount volumes twice?

Reply:
No, session scoped dependencies mean only once per run.